_data/simple-icons.json

```
@@ -2409,6 +2409,16 @@
             "hex": "AAFF89",
             "source": "https://www.channel4.com"
         },
+        {
+            "title": "Charles",
+            "hex": "F3F5F5",
+            "source": "https://www.charlesproxy.com",
+            "aliases": {
+                "aka": [
+                    "Charles Proxy"
+                ]
+            }
```

```suggestion
            "title": "Charles Proxy",
            "hex": "F3F5F5",
            "source": "https://www.charlesproxy.com"
```

We can save ourselves a redundant alias here :)